### ISSUE TITLE
Pipenv support 

### ISSUE THREAD

**Issue by jairojair** [rougeth/bottery#7]
Hi  @rougeth 

What 'u think add support to pipenv?

https://github.com/kennethreitz/pipenv

**Comment by rougeth** [comment]
@jairojair I never used pipenv and I know just a bit about it. But actually I know there is a lot of people talking about it. Do you the benefits?

**Comment by rougeth** [comment]
@jairojair I'm probably going to support it, I'm just focusing on tests and docs right now, so this will take a bit to be done.

**Comment by gabrielecker** [comment]
May I work on this?

**Comment by rougeth** [comment]
@gabrielecker sure!

**Comment by gabrielecker** [comment]
So just to be clear, by supporting pipenv you mean just adding its dependencies files?

**Comment by rougeth** [comment]
@gabrielecker actually it's probably just the `Pipfile` integrated to `setup.py`. @jairojair what did you have in mind?

**Comment by rougeth** [comment]
@gabrielecker I'm sorry, like I said, I know just a bit about pipenv, pipfile, etc and its benefits. Maybe we should discuss it more, what do you have in mind too?

**Comment by gabrielecker** [comment]
I've been searching about `setup.py` and `Pipfile ` integration and it looks like an anti-pattern.
There are a few statements about the own pipenv creator above:

> tl;dr: your setup.py is broken. setup.py should not import pipenv. that's horrible practice.

> e.g. a pipfile in that repo is for reproducing a dev environment, the setup.py is for installing the package itself.

https://github.com/kennethreitz/pipenv/issues/571

**Comment by rougeth** [comment]
@gabrielecker I'm sorry for the delay (just saw your lighting talk on #pybr13 hahaha congrats!). Well, let's close this issue, right?
